nitime/utils.py

```diff
@@ -881,7 +881,7 @@ def detect_lines(s, tapers, p=None, **taper_kws):
 
 # If we can get it, we want the cythonized version
 try:
-    from _utils import tridisolve
+    from ._utils import tridisolve
```

Comment:
Would the absolute import version of this work for you?

```suggestion
    from nitime._utils import tridisolve
```

Reply:
Yes, it works fine too